mail-filter/rspamd/rspamd-2.1.ebuild

```diff
@@ -44,6 +44,7 @@ DEPEND="${RDEPEND}"
 src_prepare() {
 	cmake-utils_src_prepare
 
+	sed -i -e '/PROJECT/s/LANGUAGES C ASM/LANGUAGES C CXX ASM/' CMakeLists.txt
```

Comment:
All external commands needs to `|| die`, you can see multiple examples in the ebuilds already :)

Reply:
Thanks and sorry you had to write it again. I always forgot :(